openfisca_uk_data/datasets/frs/frs.py

```diff
@@ -617,11 +618,41 @@ def add_benefit_income(
     frs["SSP"] = person.SSPADJ * 52
     frs["SMP"] = person.SMPADJ * 52
 
-    frs["student_loans"] = person.TUBORR
+    frs["student_loans"] = np.maximum(person.TUBORR, 0)
 
-    frs["student_payments"] = person[["ADEMAAMT", "CHEMAAMT", "ACCSSAMT"]].sum(
-        axis=1
-    ) * 52 + person[["GRTDIR1", "GRTDIR2"]].sum(axis=1)
+    frs["adult_ema"] = (
```

Comment:
Not sure these are the appropriate variable names but something like this
```suggestion
    def fill_with_mean(code, amount):
        needs_fill = (person[code] == 1) & (person[amount] < 0)
        fill_mean = person[amount][needs_fill].mean()
        filled_values = np.where(needs_fill, fill_mean, person[code])
        return np.maximum(filled_values, 0) * 52
        
    
    frs["adult_ema"] = fill_with_mean("ADEMA", "ADEMAAMT")
    frs["child_ema"] = fill_with_mean("CHEMA", "CHEMAAMT")
```

Reply:
Sorry, what do you mean by the third argument here?

Reply:
Without knowing what the columns represent, I'd expect that, since `fill_mean` takes `amount.mean()`, that we'd want `filled_values = np.where(..., ..., person[amount])` instead of `np.where(..., ..., person[code])`

Reply:
Oh you're right, fixed thanks